samples/gibbs_ensemble/run_sim.py

```diff
@@ -0,0 +1,517 @@
+#
```

Comment:
These new files need to be tested in CI. The file to update is `testsuite/scripts/samples/test_gibbs_ensemble.py`.

Reply:
The `testsuite/scripts/samples/test_gibbs_ensemble.py` was looking for the old gibbs ensemble scripts, I updated the code to run the right code in 6ca1ba0.